### ISSUE TITLE
 train-acc only increase in first epoch

### ISSUE THREAD

**Issue by maxpy** [tornadomeet/mxnet-face#9]
I compile Mxnet face version ,and train with run.sh ,but  train-acc only increase in first epoch .

INFO:root:Namespace(batch_size=256, data_dir='D:\mxnet_train\casiaa_alfred_aligned', gpus='0', kv_store='local', lr=0.0001, model_load_epoch=1, model_load_prefix='D:\mxnet_train\model\0525\alfred_aligned', model_save_prefix='D:\mxnet_train\model\0525\alfred_aligned', num_examples=432815, retrain=True)
INFO:root:Start training with [gpu(0)]
INFO:root:Epoch[0] Batch [100]  Speed: 336.65 samples/sec   Train-accuracy=0.645898
INFO:root:Epoch[0] Batch [200]  Speed: 331.79 samples/sec   Train-accuracy=0.663164
INFO:root:Epoch[0] Batch [300]  Speed: 332.34 samples/sec   Train-accuracy=0.668242
INFO:root:Epoch[0] Batch [400]  Speed: 335.01 samples/sec   Train-accuracy=0.693711
INFO:root:Epoch[0] Batch [500]  Speed: 334.37 samples/sec   Train-accuracy=0.699023
INFO:root:Epoch[0] Batch [600]  Speed: 332.04 samples/sec   Train-accuracy=0.712734
INFO:root:Epoch[0] Batch [700]  Speed: 332.06 samples/sec   Train-accuracy=0.719727
INFO:root:Epoch[0] Batch [800]  Speed: 332.64 samples/sec   Train-accuracy=0.737969
INFO:root:Epoch[0] Batch [900]  Speed: 332.11 samples/sec   Train-accuracy=0.748594
INFO:root:Epoch[0] Batch [1000] Speed: 333.26 samples/sec   Train-accuracy=0.759297
INFO:root:Epoch[0] Batch [1100] Speed: 333.02 samples/sec   Train-accuracy=0.779766
INFO:root:Epoch[0] Batch [1200] Speed: 332.66 samples/sec   Train-accuracy=0.787852
INFO:root:Epoch[0] Batch [1300] Speed: 333.59 samples/sec   Train-accuracy=0.802188
INFO:root:Epoch[0] Batch [1400] Speed: 331.93 samples/sec   Train-accuracy=0.808555
INFO:root:Epoch[0] Batch [1500] Speed: 334.06 samples/sec   Train-accuracy=0.828867
INFO:root:Epoch[0] Batch [1600] Speed: 331.78 samples/sec   Train-accuracy=0.846172
INFO:root:Update[1691]: Change learning rate to 7.00000e-05
INFO:root:Epoch[0] Resetting Data Iterator
INFO:root:Epoch[0] Time cost=1301.951
INFO:root:Saved checkpoint to "D:\mxnet_train\model\0525\alfred_aligned-0001.params"
INFO:root:Epoch[1] Batch [100]  Speed: 335.75 samples/sec   Train-accuracy=0.394023
INFO:root:Epoch[1] Batch [200]  Speed: 333.30 samples/sec   Train-accuracy=0.404844
INFO:root:Epoch[1] Batch [300]  Speed: 332.97 samples/sec   Train-accuracy=0.392188
INFO:root:Epoch[1] Batch [400]  Speed: 333.99 samples/sec   Train-accuracy=0.405156
INFO:root:Epoch[1] Batch [500]  Speed: 333.78 samples/sec   Train-accuracy=0.402422
INFO:root:Epoch[1] Batch [600]  Speed: 332.24 samples/sec   Train-accuracy=0.411250
INFO:root:Epoch[1] Batch [700]  Speed: 333.50 samples/sec   Train-accuracy=0.414883
INFO:root:Epoch[1] Batch [800]  Speed: 332.92 samples/sec   Train-accuracy=0.423516
INFO:root:Epoch[1] Batch [900]  Speed: 332.78 samples/sec   Train-accuracy=0.423867
INFO:root:Epoch[1] Batch [1000] Speed: 330.80 samples/sec   Train-accuracy=0.432031
INFO:root:Epoch[1] Batch [1100] Speed: 334.11 samples/sec   Train-accuracy=0.439883
INFO:root:Epoch[1] Batch [1200] Speed: 334.03 samples/sec   Train-accuracy=0.447852
INFO:root:Epoch[1] Batch [1300] Speed: 333.42 samples/sec   Train-accuracy=0.456641
INFO:root:Epoch[1] Batch [1400] Speed: 334.37 samples/sec   Train-accuracy=0.454023
INFO:root:Epoch[1] Batch [1500] Speed: 333.05 samples/sec   Train-accuracy=0.464219
INFO:root:Epoch[1] Batch [1600] Speed: 332.09 samples/sec   Train-accuracy=0.476289
INFO:root:Epoch[1] Resetting Data Iterator
INFO:root:Epoch[1] Time cost=1298.563
INFO:root:Saved checkpoint to "D:\mxnet_train\model\0525\alfred_aligned-0002.params"
INFO:root:Epoch[2] Batch [100]  Speed: 337.55 samples/sec   Train-accuracy=0.372266
INFO:root:Epoch[2] Batch [200]  Speed: 333.14 samples/sec   Train-accuracy=0.380742
INFO:root:Epoch[2] Batch [300]  Speed: 333.10 samples/sec   Train-accuracy=0.367578
INFO:root:Epoch[2] Batch [400]  Speed: 331.97 samples/sec   Train-accuracy=0.375937
INFO:root:Epoch[2] Batch [500]  Speed: 333.65 samples/sec   Train-accuracy=0.376563
INFO:root:Epoch[2] Batch [600]  Speed: 333.32 samples/sec   Train-accuracy=0.379492
INFO:root:Epoch[2] Batch [700]  Speed: 332.77 samples/sec   Train-accuracy=0.375469
INFO:root:Epoch[2] Batch [800]  Speed: 332.36 samples/sec   Train-accuracy=0.381328
INFO:root:Epoch[2] Batch [900]  Speed: 333.27 samples/sec   Train-accuracy=0.380625
INFO:root:Epoch[2] Batch [1000] Speed: 332.74 samples/sec   Train-accuracy=0.382031
INFO:root:Epoch[2] Batch [1100] Speed: 332.61 samples/sec   Train-accuracy=0.382539
INFO:root:Epoch[2] Batch [1200] Speed: 331.58 samples/sec   Train-accuracy=0.386680
INFO:root:Epoch[2] Batch [1300] Speed: 333.37 samples/sec   Train-accuracy=0.392227
INFO:root:Epoch[2] Batch [1400] Speed: 332.36 samples/sec   Train-accuracy=0.392734
INFO:root:Epoch[2] Batch [1500] Speed: 332.56 samples/sec   Train-accuracy=0.389766
INFO:root:Epoch[2] Batch [1600] Speed: 333.80 samples/sec   Train-accuracy=0.398555
INFO:root:Epoch[2] Resetting Data Iterator
INFO:root:Epoch[2] Time cost=1298.782
INFO:root:Saved checkpoint to "D:\mxnet_train\model\0525\alfred_aligned-0003.params"
INFO:root:Epoch[3] Batch [100]  Speed: 334.17 samples/sec   Train-accuracy=0.371758
INFO:root:Epoch[3] Batch [200]  Speed: 333.79 samples/sec   Train-accuracy=0.381953
INFO:root:Epoch[3] Batch [300]  Speed: 332.77 samples/sec   Train-accuracy=0.371523
INFO:root:Epoch[3] Batch [400]  Speed: 330.89 samples/sec   Train-accuracy=0.379844
INFO:root:Epoch[3] Batch [500]  Speed: 332.51 samples/sec   Train-accuracy=0.377305
INFO:root:Epoch[3] Batch [600]  Speed: 332.82 samples/sec   Train-accuracy=0.382734
INFO:root:Epoch[3] Batch [700]  Speed: 333.12 samples/sec   Train-accuracy=0.379844
INFO:root:Epoch[3] Batch [800]  Speed: 332.58 samples/sec   Train-accuracy=0.384102
INFO:root:Epoch[3] Batch [900]  Speed: 330.25 samples/sec   Train-accuracy=0.385234

**Comment by tornadomeet** [comment]
hello, why i see `retrain=True`here.

**Comment by maxpy** [comment]
continue train with:  model_load_epoch=1, model_load_prefix='D:\mxnet_train\model\0525\alfred_aligned'

**Comment by tornadomeet** [comment]
which script do you use? do you change any code or parameter?post it somewhere. the log you train is not normal.

**Comment by csgcmai** [comment]
I just find sth. strange with the different batch size. After changing the batch size to 256, I find that the training accuracy will decrease after first several epochs. I have not debugged it because  of the time and just for your information.

**Comment by tornadomeet** [comment]
if you change the batch size, you'd better also changed the lr value~
